plugins/out_s3/s3.c

```diff
@@ -83,6 +83,24 @@ int s3_plugin_under_test()
     return FLB_FALSE;
 }
 
+struct flb_aws_header *create_canned_acl_header(char *canned_acl)
```

Comment:
define this function with the `static` keyword since it can be private:

```
static struct flb_aws_header *create_canned_acl_header(char *canned_acl)
```

Reply:
actually never mind, I realized you use this in two files- ignore this comment